Solution1/DataAccessLayer/Repository/WetherRepository.cs

```diff
@@ -0,0 +1,30 @@
+﻿using DataAccessLayer.Abstract;
+using DataAccessLayer.Model;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataAccessLayer.Repository
+{
+    public class WetherRepositor : IWetherRepository
```

Comment:
Wether - fix misspellings here and in other places

Reply:
+